website/docs/cloud-docs/api-docs/index.mdx

```diff
@@ -48,11 +48,12 @@ HashiCorp provides a [stability policy](/terraform/cloud-docs/api-docs/stability
 
 All requests must be authenticated with a bearer token. Use the HTTP header `Authorization` with the value `Bearer <token>`. If the token is absent or invalid, HCP Terraform responds with [HTTP status 401][401] and a [JSON API error object][]. The 401 status code is reserved for problems with the authentication token; forbidden requests with a valid token result in a 404.
 
-There are three kinds of tokens available:
+There are four kinds of tokens available:
 
 - [User tokens](/terraform/cloud-docs/users-teams-organizations/users#api-tokens) — each HCP Terraform user can have any number of API tokens, which can make requests on their behalf.
 - [Team tokens](/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens) — each team can have one API token at a time. This is intended for performing plans and applies via a CI/CD pipeline.
 - [Organization tokens](/terraform/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens) — each organization can have one API token at a time. This is intended for automating the management of teams, team membership, and workspaces. The organization token cannot perform plans and applies.
+- [Audit trails tokens](/terraform/cloud-docs/users-teams-organizations/api-tokens#audit-trails-tokens) - each organization can have one audit trails token at a time. This token provides real-only access to an organization's audit trails, and is intended to be used in the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
```

Comment:
I'm not sure what "real-only" means here, and I think the sentence reads well without it, so I suggest removing. I only took a shot at rephrasing the last sentence to be a bit more app-agnostic in case we decide to use it with other apps in the future:
```suggestion
- [Audit trails token](/terraform/cloud-docs/users-teams-organizations/api-tokens#audit-trails-tokens) - each organization can have a single token that can read that organization's audit trails. Use this token type to authenticate integrations pulling audit trail data, for example, using the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
```